addons/udes_stock/models/stock_picking_type.py

```diff
@@ -244,6 +244,7 @@ class StockPickingType(models.Model):
                 "Exactly Match The Move Line Destination Location",
             ),
             ("by_products", "By Products"),
+            ("by_product_lot", "By Product and Batch Number"),
```

Comment:
Should this be Batch Number or Lot Number? I know they're interchangeable but I've seen a few instances of lot number on the desktop UI but not batch.

Reply:
We should check this with PO.

Reply:
Spoke to product and will change all references of batch to lot